build.gradle

```diff
@@ -256,8 +256,12 @@ configure(javaCodeCheckedProjects) {
         testImplementation platform('org.spockframework:spock-bom:2.1-groovy-3.0')
         testImplementation 'org.spockframework:spock-core'
         testImplementation 'org.spockframework:spock-junit4'
-        testImplementation("org.mockito:mockito-core:3.8.0")
-        testImplementation("org.mockito:mockito-inline:3.8.0")
+        if ('8'.equals(findProperty("javaVersion"))) {
+            testImplementation("org.mockito:mockito-core:4.6.1")
+            testImplementation("org.mockito:mockito-inline:4.6.1")
```

Comment:
Just curious why `mockito-inline` is not needed for 5.11.  Was it integrated into core?

Reply:
I don't know, but I confirmed with experiments that it's needed when 4.6.1 is used, and is not needed when 5.11 is used.